### DISCUSSION TITLE
Request for dbeaver spatial viewer to load raster map image

### DISCUSSION THREAD

**Discussion by tanjunwu** [dbeaver/dbeaver#20848] [Q&A] [answered]
HI, i current using dbeaver to store my geometry data offline and as it is not connected to internet, i could not load the map in the spatial viewer. however, i do have a raster world map (in .xml). 

Hence, i am wondering if i can or is it possible to load the raster world map into dbeaver to load offline?

**Comment by ShadelessFox** [comment, marked as answer]
Hello @tanjunwu,

Can you please provide more information about raster world maps? I can't find any information online.

If tiles follow the naming convention `tile-x-y-z` or something like that [you can specify your own tile layer definition](https://github.com/dbeaver/dbeaver/wiki/Working-with-Spatial-GIS-data#tile-layer-management).

Example definition:

```
'file:///C:/path/to/tiles/tile-{z}-{x}-{y}.png', {maxZoom: 18}
```

**Comment by tanjunwu** [comment]
Alright! Thanks so much!! it worked!!
